src/mscorlib/shared/System/Globalization/DateTimeParse.cs

```diff
@@ -2493,7 +2493,7 @@ internal static bool TryParse(ReadOnlySpan<char> s, DateTimeFormatInfo dtfi, Dat
         {
             if (s.Length == 0)
             {
-                result.SetFailure(ParseFailureKind.Format, nameof(SR.Format_BadDateTime), null);
+                result.SetFailure(ParseFailureKind.FormatWithParameter, nameof(SR.Format_BadDateTime), string.Empty);
```

Comment:
another Empty to remove?

Reply:
actually there are a bunch, they seem like they could call the 2 parameter overload